configurations/ca-staging.ts

```diff
@@ -14,812 +14,804 @@
  * along with this program.  If not, see https://www.gnu.org/licenses/.
  */
 
-import {
```

Comment:
I can't read this diff very well because of the indentation changes - could you perhaps comment where the real changes are? Or is it all indentation changes?

Reply:
@stephenhand, for the ca-staging.ts file, there are no changes made to it. Just linting errors that were fixed.